application/config/module.config.php

```diff
@@ -434,32 +434,32 @@
-            'Omeka\Form\SettingForm' => Service\Form\SettingFormFactory::class,
-            'Omeka\Form\ModuleStateChangeForm' => Service\Form\ModuleStateChangeFormFactory::class,
-            'Omeka\Form\SiteForm' => Service\Form\SiteFormFactory::class,
-            'Omeka\Form\SiteSettingsForm' => Service\Form\SiteSettingsFormFactory::class,
-            'Omeka\Form\Element\ResourceSelect' => Service\Form\Element\ResourceSelectFactory::class,
-            'Omeka\Form\Element\ResourceClassSelect' => Service\Form\Element\ResourceClassSelectFactory::class,
-            'Omeka\Form\Element\PropertySelect' => Service\Form\Element\PropertySelectFactory::class,
-            'Omeka\Form\Element\ItemSetSelect' => Service\Form\Element\ItemSetSelectFactory::class,
-            'Omeka\Form\Element\SiteSelect' => Service\Form\Element\SiteSelectFactory::class,
-            'Omeka\Form\Element\LocaleSelect' => Service\Form\Element\LocaleSelectFactory::class,
-            'Omeka\Form\Element\Recaptcha' => Service\Form\Element\RecaptchaFactory::class,
-            'Omeka\Form\Element\HtmlTextarea' => Service\Form\Element\HtmlTextareaFactory::class,
-            'Omeka\Form\Element\Ckeditor' => Service\Form\Element\CkeditorFactory::class,
-            'Omeka\Form\Element\CkeditorInline' => Service\Form\Element\CkeditorInlineFactory::class,
+            Form\ResourceForm::class => Service\Form\ResourceFormFactory::class,
+            Form\ResourceBatchUpdateForm::class => Service\Form\ResourceBatchUpdateFormFactory::class,
+            Form\UserForm::class => Service\Form\UserFormFactory::class,
+            Form\SettingForm::class => Service\Form\SettingFormFactory::class,
+            Form\ModuleStateChangeForm::class => Service\Form\ModuleStateChangeFormFactory::class,
+            Form\SiteForm::class => Service\Form\SiteFormFactory::class,
+            Form\SiteSettingsForm::class => Service\Form\SiteSettingsFormFactory::class,
+            Form\Element\ResourceSelect::class => Service\Form\Element\ResourceSelectFactory::class,
+            Form\Element\ResourceClassSelect::class => Service\Form\Element\ResourceClassSelectFactory::class,
+            Form\Element\PropertySelect::class => Service\Form\Element\PropertySelectFactory::class,
+            Form\Element\ItemSetSelect::class => Service\Form\Element\ItemSetSelectFactory::class,
+            Form\Element\SiteSelect::class => Service\Form\Element\SiteSelectFactory::class,
+            Form\Element\LocaleSelect::class => Service\Form\Element\LocaleSelectFactory::class,
+            Form\Element\Recaptcha::class => Service\Form\Element\RecaptchaFactory::class,
+            Form\Element\HtmlTextarea::class => Service\Form\Element\HtmlTextareaFactory::class,
+            Form\Element\Ckeditor::class => Service\Form\Element\CkeditorFactory::class,
+            Form\Element\CkeditorInline::class => Service\Form\Element\CkeditorInlineFactory::class,
```

Comment:
Oh, and I didn't notice this my first time through either... I don't know about these that switch the _keys_ to using the `::class` construction... even though these are equivalent to their class names here, I'm not sure we want to be doing that. `::class` on the _values_, since they actually are used as class names by the system, are OK/desirable (as in the other changes to this file), but I don't think we want to do this key-side change, at least not without thinking it over some more.

Reply:
Ok, I reverted it in last commit.